interface/forms/eye_mag/a_issue.php

```diff
@@ -590,7 +590,7 @@ function negate_radio(section) {
     </script>
     <!-- Add Font stuff for the look and feel.  -->
 
-    <?php Header::setupHeader(['datetime-picker', 'jquery-ui', 'jquery-ui-excite-bike', 'purecss', 'shortcut', 'opener']); ?>
+    <?php Header::setupHeader(['datetime-picker', 'purecss', 'shortcut', 'opener']); ?>
     <link rel="stylesheet" href="../../forms/<?php echo $form_folder; ?>/css/style.css" type="text/css" />
```

Comment:
Relative paths issue here -- should be using root globals variable to prevent pathing issues. It's either gonna be $GLOBALS['rootdir'] or $include_root @bradymiller which one is it